mne_bids/write.py

```diff
@@ -1035,9 +1040,15 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
     if trans is None:
         return anat_dir
 
-    if not isinstance(trans, mne.transforms.Transform):
-        raise ValueError('`trans` must be a "Transform" but is of type "{}"'
-                         .format(type(trans)))
+    if not(isinstance(trans, mne.transforms.Transform) or
+           isinstance(trans, str)):
+        raise ValueError('`trans` must be of type "Transform" or "str", '
+                         'but is of type "{}"'.format(type(trans)))
+    if isinstance(trans, str):
+        if not op.exists(trans):
+            raise ValueError('Expected `trans` to point to a *.trans file, '
+                             'but it does not exist: "{}"'.format(trans))
```

Comment:
why don't you use [`_get_trans`](https://github.com/mne-tools/mne-python/blob/897c3b2d12d0562f888784c5e18c2da31ed27ceb/mne/transforms.py#L433) ?

Reply:
wonderful, thanks!